testsuite/features/profiles/github_runner/Docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM localhost:5002/opensuse/leap:15.4
```

Comment:
From what I see, our docker buildhost `dockerfiles/buildhost/Dockerfile` use leap:15.5, so I guess we need to align those ones with it.

![image](https://github.com/user-attachments/assets/16c97ccf-f6a2-4d58-93c9-0d78318ded46)

Reply:
Slack discussion:

https://suse.slack.com/archives/C02D12TNYLS/p1740567904734179?thread_ts=1740567592.760599&cid=C02D12TNYLS

It seems we are not forced to align them, just for Kiwi images.